mzLib/Proteomics/Protein/Protein.cs

```diff
@@ -78,14 +78,14 @@ public Protein(string sequence, string accession, string organism = null, List<T
 
         /// <summary>
         /// Protein construction that clones a protein but assigns a different base sequence
-        /// For use in SILAC experiments and in decoy construction
+        /// For use in SILAC experiments
         /// </summary>
```

Comment:
please put the summary comment back to its original state and remove silacAccession from params of summary comment. All other changes here were the result of a merge